scripts/start.js

```diff
@@ -66,62 +69,90 @@ function clearConsole() {
 }
 
 var compiler = webpack(config, handleCompile);
-compiler.plugin('invalid', function () {
-  clearConsole();
-  console.log('Compiling...');
-});
-compiler.plugin('done', function (stats) {
-  clearConsole();
-  var hasErrors = stats.hasErrors();
-  var hasWarnings = stats.hasWarnings();
-  if (!hasErrors && !hasWarnings) {
-    console.log(chalk.green('Compiled successfully!'));
-    console.log();
-    console.log('The app is running at http://localhost:3000/');
-    console.log();
-    return;
-  }
+detect(PORT, (error, _port) => {
```

Comment:
`PORT` and `_port` are not very descriptive, and `_` is unnecessary.
Something like `DEFAULT_PORT` and `suggestedPort` would be better.